### DISCUSSION TITLE
Should I invole sentry for react native when I add react native as a module in my existing app which already uses sentry?

### DISCUSSION THREAD

**Discussion by Anonymous-fe** [getsentry/sentry-react-native#3098] [Q&A]
I already have an native app using sentry, I am adding react native into my app to implement some of my features. Should I add react native sentry into my project? Will it be conflict with my native sentry?

I am using the latest react native 0.71.8 version

**Comment by krystofwoldrich** [comment]
Hi, thank you for the question,
yes you can add `@sentry/react-native` to your project, but since you already have sentry setup in the native app and I assume you would like to keep it, you will need to disable the `autoInitializeNativeSdk: false`, since your native code already does it.

Depending on your setup you might need to disable some features in the RN SDK or native SDK to avoid duplicate data. For example, performance is recommended to have enabled only in one SDK not to get duplicate transactions from both layers.

There are more details on how to in our documentation. https://docs.sentry.io/platforms/react-native/manual-setup/native-init/
